src/app/components/UserAssets/index.tsx

```diff
@@ -331,15 +336,22 @@ const AssetRow: React.FC<IAssetRowProps> = ({
             <button
               className={styles.actionLink}
               onClick={() => onConvert(item.asset)}
+              disabled={tokens === '0'}
```

Comment:
You use this condition (`tokens === '0'`) on many places, extract it to a variable (e.g. `hasAnyTokens`) and use `useMemo`.